README.md

```diff
@@ -325,6 +325,18 @@ val otherRows = cachedDF.select("word_count")
 
 You can also manually specify the `filter` option, which will override automatic pushdown and Spark will do the rest of the filtering in the client.
 
+### Partitioned Tables
+
+The pseudo columns \_PARTITIONDATE and \_PARTITIONTIME are not part of the table schema. Therefore in order to query by the partitions of [partitioned tables](https://cloud.google.com/bigquery/docs/partitioned-tables) do not use the where() method shown above. Instead, add a filter option in the following manner:
```

Comment:
We should probably identify partitioned tables and fail the query instead of doing a full table scan (and add a opt-in option to full table scan if users must do it).